### DISCUSSION TITLE
Incorrect Monthly Billing for Copilot Membership

### DISCUSSION THREAD

**Discussion by AIAgentForBetterLife** [community/community#131604] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My account username is AIAgentForBetterLife. I have subscribed to the Copilot membership, which is supposed to cost $10 per month. However, I have noticed that I am being charged $20 each month.

Could you please look into this issue and provide an explanation for the discrepancy?

Attached is a screenshot for your reference.

Thank you for your assistance.

**Comment by davevad93** [comment, marked as answer]
Hi @AIAgentForBetterLife ,

I don't know if you will get an answer here in the discussions, I think you should contact directly support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or recepits` and describe your issue.

**Reply by AIAgentForBetterLife** [reply]
Thank you. I submitted it a few minutes ago.

**Reply by davevad93** [reply]
You're welcome, I hope you'll solve it! 👍

**Reply by AmalaKrishnan** [reply]
Hi.. Did you issue got resolved?

**Comment by khongyang-beep** [comment]
Why can't I chat with my Copilot, even though I just paid for it on August 5, 2025, for one month? It says: Access Denied. Its shown : Active subscription Copilot Pro Monthly payment $10.00 Next payment Aug 29, 2025. I paid for a month,  its just completed within the 2 days . Do you think is Ok?

**Comment by hayesfbg** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> My account username is AIAgentForBetterLife. I have subscribed to the Copilot membership, which is supposed to cost $10 per month. However, I have noticed that I am being charged $20 each month.
> 
> Could you please look into this issue and provide an explanation for the discrepancy?
> 
> Attached is a screenshot for your reference.
> 
> Thank you for your assistance.

**Comment by ral-it** [comment]
Hello,
I should have bought Copilot Pro $100; but I made wrong payment for Github Pro $48. So please support to process refund for $48 and I will process payment for Github Pro $48 for replacement. 
Thank you

**Comment by debasistripathy01** [comment]
YOU GUYS ARE ONE OF THE FUCKING RETARDS. YOU KEEP DEDUCTING MY MONEY $1 every fucking time but you Dont deduct it for the premium subscrption $10 at a time. Why tf do you do that . Its been 10 fuking times you deducted $1 each time and didn't even priovide premium sub. Now either refund my money or deduct rest amount and provide me Copilot sub for 1 month. Retards

**Comment by 1ca32b** [comment]
Attempt_Exceed_Limitation - Attempt exceed the limitation, refresh page to try again.

I cant save billing information says this.

**Comment by sprakash3012** [comment]
I have a $10 monthly subscription to Copilot Pro, but when I check Overview, it says $14.82. Why does my invoicing suddenly show $14.82 when I only enrolled to Copilot Pro and paid $10 last month?. Pls provide me a clarification

**Comment by Raghavcse** [comment]
2026-03-10 15:22:04.828 [error] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token. reason: NotAuthorized"

**Comment by divya-nsh** [comment]
There payment and subscription system is entire crap

**Comment by VScode-ai-nes** [comment]
<img width="897" height="644" alt="สกรีนช็อต 2026-04-25 195729" src="https://github.com/user-attachments/assets/d379fe72-d358-4386-b337-7c1a6fed0940" />
<img width="912" height="774" alt="สกรีนช็อต 2026-04-25 195808" src="https://github.com/user-attachments/assets/35e88cd5-77ed-47e0-96df-befa00aa44fd" />
<img width="994" height="360" alt="สกรีนช็อต 2026-04-25 195907" src="https://github.com/user-attachments/assets/dd75bdff-0b07-4edc-876d-c9c845f969b6" />
<img width="1005" height="641" alt="สกรีนช็อต 2026-04-25 200314" src="https://github.com/user-attachments/assets/7e459af6-a93f-42b8-bd9a-7396d6e8e777" />
Currently, my payment method shows as Invalid – authorization hold failed, and I am unable to complete the charge.

At the same time, I would like to change my plan to Pro ($4/month, Advanced tools for private repositories).

Could you please assist me with:

Correcting the payment information so that billing is successful

Ensuring my subscription is updated to the Pro plan

I have attached the invoice (INV130868817.pdf) for your reference.

Thank you for your support.

Best regards,
Sittiwat Nuengwongsa
